bin/oref0-pump-loop.sh

```diff
@@ -87,32 +87,27 @@ main() {
             touch /tmp/pump_loop_success
             echo Completed oref0-pump-loop at $(date)
             update_display
-            run_plugins
             echo
         else
             # pump-loop errored out for some reason
             fail "$@"
         fi
+        run_plugins
```

Comment:
Moving it here won't help make it run on failures, as the fail function calls exit 1.  You'll instead need to add run_plugins after update_display there as well as above.

Reply:
Thank you. I was not aware. I have updated run_plugins to run in both places so it runs either way.

Reply:
I just tweaked the placement to be more consistent between the two cases.  LMK if you don't think that will do the right thing.

Reply:
LGTM